internal/metrics/manifests/values.go

```diff
@@ -48,6 +51,20 @@ type ConfigValue struct {
 	Labels    map[string]string `json:"labels"`
 }
 
+type UIValues struct {
+	Enabled bool         `json:"enabled"`
+	ACM     ACMValues    `json:"acm,omitempty"`
+	Perses  PersesValues `json:"perses,omitempty"`
+}
+
+type ACMValues struct {
+	Enabled bool `json:"enabled"`
+}
+
+type PersesValues struct {
+	Enabled bool `json:"enabled"`
+}
+
```

Comment:
If we want to make it as simple as possible we can also remove these and just "hard code" true in the UIPlugin.yaml in the COO chart but I'll not block on this
```suggestion
}
```